workflow/scripts/compile_xlsx_report.py

```
@@ -0,0 +1,682 @@
+    # Get input and output paths
+    vcf_snv = snakemake_obj.input.vcf_snv
+    vcf_sv = snakemake_obj.input.vcf_sv
+    vcf_cnv = snakemake_obj.input.vcf_cnv
+    output_xlsx = snakemake_obj.output.xlsx
+
+    logging.info(f"Input files: SNV VCF: {vcf_snv}, SV VCF: {vcf_sv}, CNV VCF: {vcf_cnv}\nOutput file: {output_xlsx}")
+
+    # Get params
+    filter_yaml_file = snakemake_obj.params.filter_config
+    with open(filter_yaml_file) as file:
+        filters = yaml.load(file, Loader=yaml.FullLoader)
```

Ruff (0.15.20)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)